src/ring_theory/polynomial/hermite.lean

```diff
@@ -124,5 +137,72 @@ begin
 end
 
 end coeff
+end polynomial
+
+/-! ### Lemmas about `polynomial.hermite_gauss` -/
+
+section gaussian
+
+/-- The real Gaussian function -/
+def gaussian : ℝ → ℝ := λ x, real.exp (-(x^2 / 2))
+
+lemma inv_gaussian_eq : gaussian⁻¹ = λ x, real.exp (x^2 / 2) :=
+by { ext, simp [gaussian, real.exp_neg] }
+
+lemma inv_gaussian_mul_gaussian (x : ℝ) : gaussian⁻¹ x * gaussian x = 1 :=
+by rw [inv_gaussian_eq, gaussian, ← real.exp_add, add_neg_self, real.exp_zero]
+
+lemma deriv_gaussian (x : ℝ) : deriv gaussian x = -x * gaussian x :=
+by simp [gaussian, mul_comm]
+
+lemma deriv_inv_gaussian (x : ℝ) : deriv gaussian⁻¹ x = x * gaussian⁻¹ x :=
+by simp [inv_gaussian_eq, mul_comm]
+
+lemma cont_diff_gaussian : cont_diff ℝ ⊤ gaussian :=
+((cont_diff_id.pow 2).div_const 2).neg.exp
+
+lemma cont_diff.iterated_deriv :
+∀ (n : ℕ) (f : ℝ → ℝ) (hf : cont_diff ℝ ⊤ f), cont_diff ℝ ⊤ (deriv^[n] f)
+| 0     f hf := hf
+| (n+1) f hf := cont_diff.iterated_deriv n (deriv f) (cont_diff_top_iff_deriv.mp hf).2
+
+/-- The Gaussian form of `hermite n` -/
+def hermite_gauss (n : ℕ) : ℝ → ℝ :=
```

Comment:
I don't think this definition should exist; it's equal to `(hermite n).aeval r` right?

It looks to me like the main result here is `hermite_eq_gauss`; if so, I'd suggest inclining this definition and the lemmas about it into that one proof below.

Reply:
Also, calling this `hermite_gauss` feels a bit like writing
```
def two_cos := cos 0 + cos 0
```
which of course is just a strange way to write "two"!